src/Umbraco.Cms.Api.Management/ViewModels/Template/TemplateModelBase.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Umbraco.Cms.Api.Management.ViewModels.Template;
+
+public class TemplateModelBase
+{
+    public string Name { get; set; } = string.Empty;
+
+    public string Alias { get; set; } = string.Empty;
+
+    public string? Content { get; set; }
+
+    public string? MasterTemplateAlias { get; set; }
```

Comment:
This is interesting.. I know this is also how it works today, but I wonder if we should change it to be read-only and get the information from the content? 

Today you can do all different kind of weird stuff if adding one template here, but change it in the content.

IMO we should not expect the frontend to know this